sorc/emcsfc_snow2mdl.fd/model_grid.f

```diff
@@ -575,33 +572,23 @@ subroutine read_mdl_grid_info
  deallocate (lats_mdl_temp, lons_mdl_temp)
 
  return
+ end
 
- end subroutine read_mdl_grid_info
 
+!>   clean up allocatable arrays  
+!!
+!!@ author   George Gayno          org: w/np2     date: 2005-Dec-16
```

Comment:
Messed up author tag.

!! @author George Gayno @date 2005-Dec-16

No extra spaces, no colon after date.

Reply:
Corrected author tag, spacing, and (in a previous push) the subroutine end statement